gcc/config/aarch64/cygming.h

```diff
@@ -56,37 +56,12 @@ along with GCC; see the file COPYING3.  If not see
 // #define LOCAL_LABEL_PREFIX (TARGET_64BIT ? "." : "")
 
 #include <stdbool.h>
-#ifdef __MINGW32__
-#include <stdio.h>
```

Comment:
The issue is this header not available during the libgcc build of `mingw-w64-cross-gcc` package for `aarch64-w64-mingw32` target. Only the functions moved by this PR are using this header.

Reply:
This is more likely a symptom of headers missing from your environment.

Reply:
It might but:

1.  Those lines were added by you to include `FILE` symbol for the `extern`s.
2.  The necessity to use `__MINGW32__` macro here is quite suspicious as it is not used that much throught the codebase and certainly never to import standard header.
3.  Note what is defined in [https://github.com/ZacWalk/gcc-woarm64/blob/woarm64/gcc/config/i386/cygming.h#L231](https://github.com/ZacWalk/gcc-woarm64/blob/woarm64/gcc/config/i386/cygming.h#L231) file - there aren't `extern` declarations or `FILE` symbol ever used while the `extern`s are declared in [https://github.com/ZacWalk/gcc-woarm64/blob/woarm64/gcc/config/i386/i386-protos.h#L36](https://github.com/ZacWalk/gcc-woarm64/blob/woarm64/gcc/config/i386/i386-protos.h#L36) which is being included from \*.cc files only - this must be for a reason. My theory is that the `cygming.h` file is itended to be included from the code that might not have the `stdio.h` available while the code that needs to use the actual reference to the extern functions should include `???-protos.h` somehow. This is the case for the aarch64 target as well - there is no problem finding `FILE` symbol, when the `extern` declaration with `FILE` parameter is included, e.g., from `winnt.cc`.